src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/rca/framework/util/SQLiteQueryUtils.java

```diff
@@ -40,21 +42,12 @@
  * A utility class to query cluster, node and resource level summary for a rca
  */
 public class SQLiteQueryUtils {
-  private static final Map<String, String> temperatureProfileNestedSummaryMap;
   private static final Set<String> clusterLevelRCA;
   private static final Set<String> temperatureProfileRCASet;
 
-  static {
```

Comment:
we are changing this at the same time. I think we can reuse what is already there. Let's discuss this offline